src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/AddClientCertRunner.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.Commands
+{
+    public class AddClientCertRunner
+    {
+        public static void Run(AddClientCertArgs args, Func<ILogger> getLogger)
```

Comment:
1. Keep existing standards.

```suggestion
        public static void Run(AddClientCertArgs args, ILogger getLogger)
```

Or, 

2. Try to minimize calls to get a logger instance. `Lazy<ILogger>` could help.

```suggestion
        public static void Run(AddClientCertArgs args, Lazy<ILogger> getLogger)
```

Reply:
Cannot be achieved because of XPlat automatic code usage generation. And there second variable MUST be Func<ILogger>